rapid_response_xblock/static/html/rapid.html

```diff
@@ -1,6 +1,8 @@
 <div class="rapid-response-block" data-staff="{{ is_staff }}" data-open="{{ is_open }}">
   <div id="rapid-response-content">
   </div>
+  <div id="rapid-response-results">
+  </div>
```

Comment:
I just realized that we can't use id's for `rapid-response-content` or `rapid-response-results`. We can have multiple rapid response problems rendered on a single page. As-is, all graphs on a given page are rendered into the first `#rapid-response-results` element instead of rendering under their corresponding problem

Reply:
Right... I'll investigate how to fix this